packages/table/src/table.tsx

```diff
@@ -826,6 +830,33 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         return this.maybeRenderRegions(styler);
     }
 
+    private maybeRenderMenuRegions() {
+        const styler = (region: IRegion): React.CSSProperties => {
+            const { grid } = this;
+            const { viewportRect } = this.state;
+            if (viewportRect == null) {
+                return {};
+            }
+            const cardinality = Regions.getRegionCardinality(region);
+            const style = grid.getRegionStyle(region);
+
+            switch (cardinality) {
+                case RegionCardinality.FULL_TABLE:
+                    style.right = "0px";
+                    style.bottom = "0px";
+                    style.top = "0px";
+                    style.left = "0px";
```

Comment:
use numeral `0` instead of string 
https://stylelint.io/user-guide/rules/length-zero-no-unit/